### PR TITLE
Printer paper tray status and error states

### PR DESCRIPTION
**Pull request by sjtarik**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
Looks pretty good overall @sjtarik 

Can we try to add test data for these?

File: includes/discovery/toner.inc.php [hunk 1]
```diff
@@ -58,6 +58,38 @@
...
+    $tray_oids = snmpwalk_cache_oid($device, 'prtInputName', [], 'Printer-MIB');
+    if (!empty($tray_oids)) {
+        $tray_oids = snmpwalk_cache_oid($device, 'prtInputCurrentLevel', $tray_oids, 'Printer-MIB');
+        $tray_oids = snmpwalk_cache_oid($device, 'prtInputMaxCapacity', $tray_oids, 'Printer-MIB');
+    }
+
+    d_echo($tray_oids);
+    foreach ($tray_oids as $index => $data) {
+        $last_index = substr($index, strrpos($index, '.') + 1);
+        $type = 'jetpapertray';
```
Comment (murrant) [inline includes/discovery/toner.inc.php]: Can probably drop jet here and just go papertray (jetdirect above is for legacy reasons)

**Comment by murrant** [comment]
Polling code looks nice, but I think it might be nice to have separate boxes for them in the webui.

**Comment by murrant** [comment]
I modified this to capture type data and display it a bit nicer in the webui:

image

**Comment by sjtarik** [comment]
that looks great @murrant. printer support needed some support and now it gets a lot of it :) great progress.

**Pull request merged**